src/HDF5.jl

```diff
@@ -931,7 +970,18 @@ end
 t_commit(parent::Union{HDF5File, HDF5Group}, path::String, dtype::HDF5Datatype) = t_commit(parent, path, dtype, p_create(H5P_LINK_CREATE))
 
 a_create(parent::Union{HDF5File, HDF5Object}, name::String, dtype::HDF5Datatype, dspace::HDF5Dataspace) = HDF5Attribute(h5a_create(checkvalid(parent).id, name, dtype.id, dspace.id), file(parent))
-p_create(class, toclose=false) = HDF5Properties(h5p_create(class), toclose)
+p_create(class, toclose=false) = HDF5Properties(h5p_create(class), toclose, class)
+function p_create(class, toclose, prop1::String, val1, pv...) 
```

Comment:
`p_create` appears to be a constructor for `HDF5Properties`, should we rename it `HDF5Properties`? Likewise for `a_create`

Reply:
Would probably be a good idea, though I didn't implement that to not break the API with my changes. I don't really know to what extent other packages rely on that feature. If this is desired renaming is done quickly.